protobuf-solidity/src/protoc/plugin/gen_sol.py

```diff
@@ -223,20 +224,21 @@ def gen_global_enum(file: FileDescriptor, delegate_codecs: List[str]):
     enum_definition = gen_enum_definition(file),
   ))
 
-SOLIDITY_NATIVE_TYPEDEFS = "SolidityTypes.proto"
 RUNTIME_FILE_NAME = "ProtoBufRuntime.sol"
 PROTOBUF_ANY_FILE_NAME = "GoogleProtobufAny.sol"
 GEN_RUNTIME = False
 COMPILE_META_SCHEMA = False
 def apply_options(params_string):
+  global GEN_RUNTIME
   params = util.parse_urllike_parameter(params_string)
   if "gen_runtime" in params:
-    global GEN_RUNTIME
     GEN_RUNTIME = True
-    name = params["gen_runtime"]
-    if name.endswith(".sol"):
-      global RUNTIME_FILE_NAME
-      RUNTIME_FILE_NAME = name
+    change_runtime_file_names(params["gen_runtime"])
+  if "use_runtime" in params:
```

Comment:
Can use option use_runtime and gen_runtime together?

Reply:
@bluele No, the previously specified option is overridden by the later specified one.

Reply:
@bluele Excuse me, the above comment was wrong.
Which option will be effective depends on the implementation details of Python.

Reply:
@siburu  ok. if that's so, it would be better to use `elif` statement instead of `if` to clarify the priority of these options.

Reply:
I'll fix it to raise an exception when both options are specified.

Reply:
> I'll fix it to raise an exception when both options are specified.

I think it's good :+1:

Reply:
done